frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -384,16 +384,18 @@ export const useApplicationsInProgressQuery = ({
   take = 10,
   searchString = "",
   sorting = [],
+  pendingPermits = false,
 }: {
-  page: number;
-  take: number;
+  page?: number;
+  take?: number;
   searchString?: string;
-  sorting: SortingConfig[];
+  sorting?: SortingConfig[];
+  pendingPermits?: boolean;
 }) => {
   return useQuery({
-    queryKey: ["applicationsInProgress", page, take, sorting],
+    queryKey: ["applicationsInProgress", page, take, sorting, pendingPermits],
     queryFn: () =>
-      getApplicationsInProgress({ page, take, searchString, orderBy: sorting }),
+      getApplicationsInProgress({ page, take, searchString, orderBy: sorting, pendingPermits }),
```

Comment:
updated the hook to convert all the parameters to optional, which makes it easier to query for the `pendingPermits` when we don't need to worry about pagination on the result...